Dockerfile

```
@@ -0,0 +1,30 @@
+FROM python:3.12-slim-bullseye as base
+
+LABEL maintainer="Togglecorp Dev"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /code
+
+COPY pyproject.toml poetry.lock /code/
+
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends \
+        # Build required packages
+        gcc libc-dev gdal-bin libproj-dev \
+        # Helper packages
+        wait-for-it \
+    # Upgrade pip and install python packages for code
+    && pip install --upgrade --no-cache-dir pip poetry \
+    && poetry --version \
+    # Configure to use system instead of virtualenvs
+    && poetry config virtualenvs.create false \
+    && poetry install --no-root \
+    # Clean-up
+    && pip uninstall -y poetry virtualenv-clone virtualenv \
```

Let's add this one too
```suggestion
    && rm -rf /root/.cache/pypoetry \
    && pip uninstall -y poetry virtualenv-clone virtualenv \
```